projects/packages/videopress/src/client/admin/components/edit-video-details/index.tsx

```diff
@@ -33,24 +37,43 @@ const Header = ( { saveDisabled = true }: { saveDisabled?: boolean } ) => {
 	);
 };
 
-const Infos = () => {
+const Infos = ( { video } ) => {
+	const [ title, setTitle ] = useState( video?.title );
+	const [ description, setDescription ] = useState( video?.description );
+	const [ caption, setCaption ] = useState( video?.caption );
+
+	useEffect( () => {
+		setTitle( video?.title );
+		setDescription( video?.description );
+		setCaption( video?.caption );
+	}, [ video ] );
+
 	return (
 		<>
-			<Input label="Title" name="title" onChange={ noop } onEnter={ noop } size="large" />
 			<Input
+				value={ title }
+				label="Title"
```

Comment:
minor one: translate.

```suggestion
				label={ __( 'Title, 'jetpack-videopress-pkg' ) }
```